sdks/python/apache_beam/runners/portability/spark_runner.py

```
@@ -93,9 +93,7 @@ def path_to_jar(self):
       return self._jar
     else:
       if self._spark_version == '2':
```

Why we still need this check?

---

yes, if somebody explicitly requires spark 2 it should fail fast